includes/definitions.inc.php

```diff
@@ -2228,6 +2228,11 @@
 $config['os'][$os]['over'][2]['graph'] = 'device_mempool';
 $config['os'][$os]['over'][2]['text']  = 'Memory Usage';
 
+$os = 'trapeze';
+$config['os'][$os]['text']             = 'Juniper Wireless';
+$config['os'][$os]['type']             = 'wireless';
+$config['os'][$os]['icon']             = 'junos';
+
```

Comment:
Sure. But i thought about adding it later on as i have plans to add more sensors such as sessions counters for each AP.